verifier/detail_template.html

```diff
@@ -615,30 +615,50 @@
         }
     }
 
+    function escapeHtml(unsafe) {
+      // To make sure that table entries won't break the HTML code.
+      if (typeof unsafe === 'string') {
+        return unsafe
+        .replace(/&/g, "&amp;")
+        .replace(/</g, "&lt;")
+        .replace(/>/g, "&gt;")
+        .replace(/"/g, "&quot;")
+        .replace(/'/g, "&#039;");
+      }
+      return unsafe;
+    }
```

Comment:
Thought: I don't see a build-in JS function for this. Seems you either write this function yourself or use the DOM. I think most people don't need this function any more because they use a templating engine.